examples/pxScene2d/src/pxShaderResource.h

```diff
@@ -85,6 +85,7 @@ class pxShaderResource: public pxResource, public shaderProgram
   bool needsFbo()   { return mNeedsFbo;   }
   int32_t passes()  { return mPasses;     }
 
+  virtual void processDownloadedResource(rtFileDownloadRequest* fileDownloadRequest);
```

Comment:
Please remove (see comment above) and use the base class version since the implementation is identical.

Reply:
Done